packages/slack/src/api/base/request.ts

```diff
@@ -3,25 +3,47 @@ import * as rp from 'request-promise-native'
 import { logger, requestToLogLevel } from '../../logger'
 
 type Callback = (arg: any) => void
-type Task = { uri: string, form: any, token: string, resolve: Callback, reject: Callback }
+type Task = { uri: string, form: any, token: string, retryable: boolean, resolve: Callback, reject: Callback }
+const DEFAULT_RETRY_AFTER = 10 * 1000 // 10 seconds
```

Comment:
```suggestion
const DEFAULT_RETRY_AFTER = 10 * 1000 // miliseconds
```

I'd suggest to rather specify unit than restating value